warehouse/integrations/datalake/schema-repository/glue.go

```diff
@@ -271,7 +273,8 @@ func (g *GlueSchemaRepository) updateTable(ctx context.Context, tableName string
 	glueInput := &glue.UpdateTableInput{
 		DatabaseName: aws.String(g.Namespace),
 		TableInput: &types.TableInput{
-			Name: aws.String(tableName),
+			Name:      aws.String(tableName),
+			TableType: aws.String("EXTERNAL_TABLE"),
```

Comment:
`TableType cannot be null` error for the customer won't be resolved for existing tables unless there is a schema change (which would trigger a call to `updateTable` method). Is that correct?

Reply:
Yes. Unless the update method is called, it won't affect.